composer.json

```diff
@@ -38,10 +38,10 @@
       "vagrant ssh -c \"cd /vagrant/content/plugins/core-sitemaps && WP_TESTS_DB_PASS=password composer run test:phpunit\""
     ],
     "test:phpcs": [
-      "vendor/bin/phpcs -nps --colors --report-code --report-summary --report-width=80 ."
+      "phpcs"
```

Comment:
I'd prefer to use the dependency version to ensure everyone is getting the same output. Should be: `vendor/squizlabs/php_codesniffer/bin/phpcs`.

Reply:
FYI: for me, running `which phpcs` returns the globally installed path in my users directory.

Reply:
This should not be needed.

https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands

Reply:
> FYI: for me, running `which phpcs` returns the globally installed path in my users directory.
You're probably running it in the terminal Joe, whereas mean to say if that line is changed in the composer.json.

Reply:
Ah yeah, misunderstood what you were suggesting. Wasn't aware that composer would prioritize a vendor version over whatever was globally available. Nice.